Resources/Prototypes/_SSS/Objects/store_items.yml

```
@@ -2,7 +2,7 @@
   parent: BoxCardboard
   id: BoxDehydratedCarp
   name: dehydrated carp box
-  description: uplink-carp-box-desc
+  description: Box filled with 4 dehydrated carps. Make sure your friends pet them as well!
```

The way its currently set up, this will ignore `uplink-carp-box-desc` set in the .ftl file.
```suggestion
  description: uplink-carp-box-desc
```

---

The item itself seems to completely ignore the locale file. It is only ever parsed for stuff like the traitor uplink menu. Not the object itself. If you check in game, the description of the entity is literally just "uplink-carp-box-desc"

---

Alright